queries/html/injections.scm

```diff
@@ -6,7 +6,50 @@
    (quoted_attribute_value (attribute_value) @css))
  (#eq? @_attr "style"))
 
-((script_element
-  (raw_text) @javascript))
-
 (comment) @comment
+
+(element
+   (self_closing_tag
+      (tag_name) @_tagname (#eq? @_tagname "input")
+      ((attribute
+          (attribute_name) @_attr
+          (quoted_attribute_value (attribute_value) @regex)))
+       (#eq? @_attr "pattern")))
+
+(element
+   (start_tag
+      (tag_name) @_tagname (#eq? @_tagname "input")
+      ((attribute
+          (attribute_name) @_attr
+          (quoted_attribute_value (attribute_value) @regex)))
+       (#eq? @_attr "pattern")))
```

Comment:
```suggestion
(element
   (_
      (tag_name) @_tagname (#eq? @_tagname "input")
      ((attribute
          (attribute_name) @_attr
          (quoted_attribute_value (attribute_value) @regex)))
       (#eq? @_attr "pattern")))
```

Reply:
great suggestion. I tried the same for unquoted attrs, but i think that's a genuine case for alternations, which appear to not be supported in these files?